ext/lib/crypto/mbedtls/Kconfig

```diff
@@ -17,6 +17,14 @@
 #
 
 
+config MBEDTLS_EXTERNAL_CONFIG
+	bool
+	help
+	  This internal option controls if mbedTLS is configured by external
```

Comment:
It does more than change the Kconfig file. It also changes the build scripts.

Reply:
Yes, I agree. I will update the text.